manage.py

```
@@ -129,5 +135,47 @@ def setup_general():
     Agency.insert_agencies()
     EditableHTML.add_default_faq()
 
+
+@manager.command
+def run_worker():
+    """Initializes a slim rq task queue."""
+    listen = ['default']
+    conn = Redis(
+        host=app.config['RQ_DEFAULT_HOST'],
+        port=app.config['RQ_DEFAULT_PORT'],
+        db=0,
+        password=app.config['RQ_DEFAULT_PASSWORD']
+    )
+
+    with Connection(conn):
+        worker = Worker(map(Queue, listen))
+        worker.work()
+
+
+@manager.command
+def run_scheduler():
+    """Initializes a rq scheduler."""
+    conn = Redis(
+        host=app.config['RQ_DEFAULT_HOST'],
+        port=app.config['RQ_DEFAULT_PORT'],
+        db=0,
+        password=app.config['RQ_DEFAULT_PASSWORD']
+    )
+
+    setup_loghandlers('INFO')
+    scheduler = Scheduler(connection=conn, interval=60.0)
+    for _ in xrange(10):
+        try:
+            scheduler.run()
+        except ValueError, exc:
+            if exc.message == 'There\'s already an active RQ scheduler':
+                scheduler.log.info(
+                    'An RQ scheduler instance is already running. Retrying in '
+                    '%d seconds.', 10,
+                )
+                time.sleep(10)
+            else:
+                raise
```

A naked raise, does this work? Should we raise the original exception `exc`?